x-pack/plugins/lens/server/routes/existing_fields.ts

```diff
@@ -113,6 +116,7 @@ async function fetchFieldExistence({
   const docs = await fetchIndexPatternStats({
     fromDate,
     toDate,
+    dslQuery: JSON.parse(dslQuery),
```

Comment:
Nitpick: You've chosen to keep this as a GET request, so the query is passed as a string. Would it make sense to turn this into a POST request with a query body?